src/pages/Languages/index.tsx

```diff
@@ -142,7 +142,13 @@ export const Languages = () => {
       title: formatMessage({ defaultMessage: 'Polish' }),
       language: 'Polskie',
     },
-  ].filter(lang => supportedLanguages.includes(lang.code));
+  ]
+    .filter(lang => supportedLanguages.includes(lang.code))
+    .sort((a, b) => {
```

Comment:
I think best approach here may be `localCompare` as it handles several edge cases with string comparison:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/localeCompare

Reply:
Good call, thanks for fixing that and for the reference